core/designsystem/src/main/java/com/mifos/core/designsystem/component/MifosProgressIndicator.kt

```diff
@@ -53,4 +65,29 @@ fun MifosCircularProgress() {
         )
     }
 
+}
+
+@Composable
+fun MifosLoadingCircularProgress() {
```

Comment:
ProgressDialog becomes depreciated because it blocks the UI. We don't need to custom build it. Keep using CircularProgressIndicator

Reply:
I have replaced progress dialog with CircularProgressIndicator